modules/mcagar/src/main/ts/ephox/mcagar/api/McEditor.ts

```diff
@@ -12,6 +12,7 @@ const pFromElement = <T extends EditorType = EditorType>(element: SugarElement<E
   return new Promise((resolve, reject) => {
     const nuSettings: Record<string, any> = {
       toolbar_mode: 'wrap',
+      promotion: false,
```

Comment:
These changes will need a McAgar changelog added and if it's not a patch release (this might be a minor? not really sure what to call it), then we'd also need to add the version to `versions.txt`.